### PR TITLE
Rakshit

### PR DESCRIPTION
**Pull request by v-raras**

### CODE REVIEW

File: CRM Solution Manager/CRM Solution/App.config [hunk 1]
```diff
@@ -1,11 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
 <configuration>
   <appSettings>
-    <add key="OrgServiceUrl" value="https://test.api.crm8.dynamics.com/XRMServices/2011/Organization.svc" />
-    <add key="DynamicsUserName" value="test@test.onmicrosoft.com" />
-    <add key="DynamicsPassword" value="*****" />
+    <add key="OrgServiceUrl" value="https://msdyn.api.crm8.dynamics.com/XRMServices/2011/Organization.svc" />
+    <add key="DynamicsUserName" value="v-raras@msdyn.onmicrosoft.com" />
```
Comment (amrullah08) [inline CRM Solution Manager/CRM Solution/App.config]: you have checked in sensitive data, remove all this sensitie data

File: CRM Solution Manager/FunctionApp1/.gitignore [hunk 2]
```diff
@@ -0,0 +1,264 @@
...
+# Paket dependency manager
+.paket/paket.exe
+paket-files/
+
+# FAKE - F# Make
+.fake/
+
+# JetBrains Rider
+.idea/
+*.sln.iml
```
Comment (amrullah08) [inline CRM Solution Manager/FunctionApp1/.gitignore]: why such big gitignore file

Comment (v-raras) [reply CRM Solution Manager/FunctionApp1/.gitignore]: this file got generated automatically at the time of azure function creation. But now, whole function app1 Project can be deleted, as it is no longer in use. We can convert it to azure function anytime in future if required, so not needed as of now.

File: CRM Solution Manager/CRM Solution/App.config (hunk 1 above)
Comment (v-raras) [reply CRM Solution Manager/CRM Solution/App.config]: committed updated app.config

**Pull request merged**